addon/mixins/authenticated-route-mixin.js

```diff
@@ -80,10 +75,27 @@ export default Mixin.create({
     @default 'login'
     @public
   */
-  authenticationRoute: computed(function() {
-    return Configuration.authenticationRoute;
+  authenticationRoute: computed({
+    get() {
+      if (this.hasOwnProperty('authenticationRouteOverride')) {
+        return this.authenticationRouteOverride;
+      } else {
+        return Configuration.authenticationRoute;
+      }
+    },
+
+    set(key, value) {
+      this.authenticationRouteOverride = value;
+      return value;
+    }
   }),
 
+  init() {
+    this._super(...arguments);
+    let owner = getOwner(this);
+    this._router = owner.lookup('service:router') || owner.lookup('router:main');
+  },
+
```

Comment:
It looked to me that in other files, `init` was placed after the various computed property definitions, but it’s easily moved if I guessed wrong!

Reply:
Yes, I think that's what we've been doing so far 👍